UselessMine.cpp

```diff
@@ -455,17 +482,39 @@ bool UselessMine::SlashCommand(int playerID, bz_ApiString command, bz_ApiString
             bz_sendTextMessage(BZ_SERVER, playerID, "Death messages reloaded");
             return true;
         }
-        else if (params->size() == 0)
+        if (params->get(1) == "defusalmessages")
```

Comment:
The way I solved this (and the matching one) at the same time was to put the check for `params->size() == 0` first, and put the other two options in an else block.